.changeset/stale-moles-nail.md

```diff
@@ -0,0 +1,31 @@
+---
+'sku': minor
+---
+
+`vite/loadable`: Add the `resolveComponent` option to the `loadable` function. The `loadable` function returns the `default` export by default. By using the `resolveComponent` function you can specify the correct component for `loadable` to use.
+
+**example usage**
+
+```typescript
+// src/MyComponent.tsx
+import React from 'react';
```

Comment:
Probably not worth including the `react` import these days. Sku consumers don't need it anymore at least.

```suggestion
```